libs/custom-paper-handler/src/driver/coders.ts

```diff
@@ -207,7 +207,7 @@ export enum ConfigureScannerOptionSensorConfig {
 export enum ConfigureScannerFlags {
   NONE = 0x00,
   SCAN_BACKWARDS = 0x01,
-  SCAN_IN_PARK = 0x02,
+  SCAN_IN_PARK = 0x03,
```

Comment:
For my understanding: this is essentially setting both the `0x01` and `0x02` flags, right?

Reply:
Correct!